website/docs/reference/snapshot-properties.md

```diff
@@ -41,3 +41,37 @@ snapshots:
 
 ```
 </File>
+
+
+## Define snapshots in YAML
+
+Defining a snapshot in YAML treats snapshots as a distinct resource type, separate from models, allowing for improved organization and consistency within your code.
+
+They don't contain logic, similar to exposures, sources, and tests. To add extra logic to your snapshot, you can break it out into an ephemeral model for cleaner development and testing, and then create a snapshot of that model.
```

Comment:
This isn't quite true. The logic they contain is `select * from relation`. Also, while exposures and sources don't have logic, tests do.